scss/boost_union/post.scss

```
@@ -21,11 +21,17 @@
     }
     /* Make sure that the back to top button is not covered by anything else. */
     z-index: $zindex-dropdown;
+    /* Animate position together with footer button. */
+    transition: right 0.2s;
```

Looking at the place where Moodle core sets this style for the footer button on https://github.com/moodle/moodle/blob/master/theme/boost/scss/moodle/layout.scss#L199, you will see that the transition is built with a SCSS include.

So, I think this line should say:

```suggestion
    @include transition(0.2s);
```